qiskit/providers/aer/backends/pulse_simulator.py

```diff
@@ -9,7 +9,7 @@
 # Any modifications or derivative works of this code must retain this
 # copyright notice, and modified files need to carry a notice indicating
 # that they have been altered from the originals.
-# pylint: disable=arguments-differ
+# pylint: disable=arguments-differ, missing-return-type-doc
```

Comment:
We shouldn't need to add this pylint skip, what functions were causing the `missing-return-type-doc` error?